cypress/e2e/login.cy.tsx

```diff
@@ -48,4 +50,60 @@ describe('Button Component', () => {
     cy.get('.modal__btn').click();
     cy.get('.modal__btn').should('not.exist');
   });
+  it('Should log in by entering the secret key and changing to the wallet path', () => {
+    cy.get('.check__input').should('exist');
+    cy.get('.check__input').should('not.be.checked');
+    cy.contains('button', 'Continuar').should('exist');
+    cy.get('.check__input').check();
+    cy.get('.check__input').should('be.checked');
+    cy.contains('button', 'Continuar').click();
+    cy.contains('label', 'TU CLAVE SECRETA').should('be.visible');
+    cy.get('input[type="password"]').type(Cypress.env('secret_key'));
+    cy.get('button[class="button button__complete"]').click();
+    cy.wait(3000);
+    cy.contains('h3', 'Su clave pública estelar').should('be.visible');
+  });
+  it('Should log in by entering the secret key and changing to the wallet path', () => {
+    cy.get('.check__input').should('exist');
+    cy.get('.check__input').should('not.be.checked');
+    cy.contains('button', 'Continuar').should('exist');
+    cy.get('.check__input').check();
+    cy.get('.check__input').should('be.checked');
+    cy.contains('button', 'Continuar').click();
+    cy.contains('label', 'TU CLAVE SECRETA').should('be.visible');
+    cy.get('input[type="password"]').type(Cypress.env('secret_key'));
+    cy.get('button[class="button button__complete"]').click();
+    cy.wait(3000);
+    cy.contains('h3', 'Su clave pública estelar').should('be.visible');
+  });
+  it.only('Should give an error message when entering an invalid secret key', () => {
```

Comment:
Why it.only?

Reply:
I was testing that test only now I remove it and try the tests again